examples/hex-game/README.md

```diff
@@ -53,18 +53,18 @@ export LINERA_STORAGE_0="rocksdb:$LINERA_TMP_DIR/client_0.db"
 export LINERA_WALLET_1="$LINERA_TMP_DIR/wallet_1.json"
 export LINERA_STORAGE_1="rocksdb:$LINERA_TMP_DIR/client_1.db"
 
+linera --with-wallet 0 wallet init --faucet $FAUCET_URL
 linera --with-wallet 1 wallet init --faucet $FAUCET_URL
-linera --with-wallet 2 wallet init --faucet $FAUCET_URL
 
-INFO_1=($(linera --with-wallet 1 wallet request-chain --faucet $FAUCET_URL))
-INFO_2=($(linera --with-wallet 2 wallet request-chain --faucet $FAUCET_URL))
+INFO_1=($(linera --with-wallet 0 wallet request-chain --faucet $FAUCET_URL))
```

Comment:
1 was intended to keep the consistency with the variable name

Reply:
Right, yes, we should either use 0 and 1 for the variables, too, or 1 and 2 for the wallets.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3738/commits/e88252d895a2911e1fc85aacd09ca93f045051f6.